schema/postgresql/t/06-get_sla_ok_percent_postgres.sql

```diff
@@ -0,0 +1 @@
+../get_sla_ok_percent_postgres.sql
```

Comment:
This is a symlink, but `../get_sla_ok_percent_postgres.sql` does not exist in this branch, it's `../get_sla_ok_percent.sql`

Reply:
Fixed